src/tests/dev.test.js

```diff
@@ -64,6 +64,49 @@ test.serial('netlify dev functions timeout', async t => {
   })
 })
 
+test('netlify dev env file', async t => {
+  let data = ""
+  await new Promise((resolve, reject) => {
```

Comment:
Small detail: using a HTTP client library like `got` might help remove some lines of code.

Reply:
One thing at a time 😊